web-app/src/app/(app)/dashboard/gallery/components/tags.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { CirclePlus } from "lucide-react";
+import { useTranslations } from "next-intl";
+import React from "react";
+
+import { Badge } from "@/components/ui/badge";
+import { Button } from "@/components/ui/button";
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+
+const ValidTags = ["Analytics", "Finance", "Trends", "Forecasting", "Data"];
+
+interface TagsProps {
+  tags: string[];
+  onChange: (tags: string[]) => void;
+}
+
+export default function Tags({ tags, onChange }: TagsProps) {
+  const t = useTranslations("App.Gallery.FilterSection");
+
+  const handleSelectTag = (tag: string, checked: boolean) => {
+    if (checked) {
+      onChange([...tags, tag]);
+    } else {
+      onChange(tags.filter((t) => t !== tag));
+    }
+  };
+
+  return (
+    <div className="flex gap-2">
+      <DropdownMenu>
+        <DropdownMenuTrigger asChild>
+          <Button
+            variant="outline"
+            className="h-12 items-center gap-2 border-dashed text-base"
```

Comment:
Why did you add `h-12`? We wanted to work with the default behavior as much as possible.
As I see the design also uses default height.